src/pages/container/EventsTable/util.tsx

```diff
@@ -41,3 +47,15 @@ export const getEventCopy = (
       return null;
   }
 };
+
+const LinkWrapper = styled.span`
+  position: relative;
+  top: 6px;
+`;
+const StyledLink = styled(Link)`
+  max-width: 300px;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  display: inline-block;
+`;
```

Comment:
I think the `LinkWrapper` styling is a little strange you can fix the alignment issues by providing a wrapper around the entire text block and giving it a flex property. 
Followed by replacing the spans that wrap each of the strings with `<DescriptionWrapper />`

```suggestion
const DescriptionWrapper = styled.div`
  display: flex;
`;
const StyledLink = styled(Link)`
  display: inline-block;
  max-width: 200px; 
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
`;

```

This accomplishes the same effect without needing to add custom positioning padding.
<img width="772" alt="image" src="https://github.com/evergreen-ci/spruce/assets/4605522/c3a7af88-2fd6-4998-a23d-bebb92c8200b">

Reply:
I think it should look correct if you remove `LinkWrapper` and use the styles from Sophie's PR i.e.

```
const StyledLink = styled(StyledRouterLink)`
  span {
    max-width: 200px;
    overflow: hidden;
    text-overflow: ellipsis;
  }
`;
```

No need to add `DescriptionWrapper` in this method